packages/mongo-nav/src/mongo-select/ProjectSelect.tsx

```diff
@@ -233,7 +233,7 @@ function ProjectSelect({
 
   const [buttonElement, setButtonElement] = useState<HTMLElement | null>(null);
 
-  const CurrentIcon = open ? CaretUpIcon : CaretDownIcon;
+  const CaretIcon = open ? CaretUpIcon : CaretDownIcon;
```

Comment:
We do this in two different ways between project and org select. Can we fix that?